<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1.2 (Python 3.9.1 on win32)" generated="20220204 23:11:12.296" rpa="false" schemaversion="2">
<suite id="s1" name="Open Website" source="C:\Users\007\PycharmProjects\test-qa-ascend-03\keyword\website\page\open_website.robot">
<test id="s1-t1" name="Open wesite">
<kw name="Open Browser">
<arg>${base_url.herokuapp}</arg>
<arg>${browser.chrome}</arg>
<msg timestamp="20220204 23:11:12.323" level="FAIL">No keyword with name 'Open Browser' found.</msg>
<status status="FAIL" starttime="20220204 23:11:12.323" endtime="20220204 23:11:12.323"/>
</kw>
<status status="FAIL" starttime="20220204 23:11:12.322" endtime="20220204 23:11:12.324">No keyword with name 'Open Browser' found.</status>
</test>
<status status="FAIL" starttime="20220204 23:11:12.298" endtime="20220204 23:11:12.326"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Open Website">Open Website</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220204 23:11:12.319" level="ERROR">Error in file 'C:\Users\007\PycharmProjects\test-qa-ascend-03\keyword\website\page\open_website.robot' on line 2: Resource file '..\..\resource\imports.robot' does not exist.</msg>
<msg timestamp="20220204 23:11:12.320" level="ERROR">Error in file 'C:\Users\007\PycharmProjects\test-qa-ascend-03\keyword\website\page\open_website.robot' on line 3: Variable file '..\..\resource\common_configs.yaml' does not exist.</msg>
</errors>
</robot>
